include/amici/abstract_model.h

```diff
@@ -48,8 +48,7 @@ class AbstractModel {
      * @param t time
      * @param x state
      * @param dx time derivative of state (DAE only)
-     * @param xdot array to which values of the residual function will be
-     * written
+     * @param xdot array to which values of the residual function will be written
```

Comment:
Intended change in line width? I liked the 80... If intended, please update in `.clang-format`.